iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HMSTablePropertyHelper.java

```diff
@@ -77,21 +94,13 @@ private HMSTablePropertyHelper() {
   public static String translateToIcebergProp(String hmsProp) {
     return ICEBERG_TO_HMS_TRANSLATION.inverse().getOrDefault(hmsProp, hmsProp);
   }
+
   /** Updates the HMS Table properties based on the Iceberg Table metadata. */
-  public static void updateHmsTableForIcebergTable(
-      String newMetadataLocation,
-      Table tbl,
-      TableMetadata metadata,
-      Set<String> obsoleteProps,
-      boolean hiveEngineEnabled,
-      long maxHiveTablePropertySize,
-      String currentLocation) {
-    Map<String, String> parameters =
-        Optional.ofNullable(tbl.getParameters()).orElseGet(Maps::newHashMap);
-    Map<String, String> summary =
-        Optional.ofNullable(metadata.currentSnapshot())
-            .map(Snapshot::summary)
-            .orElseGet(ImmutableMap::of);
+  public static void updateHmsTableForIcebergTable(String newMetadataLocation, Table tbl, TableMetadata metadata,
+      Set<String> obsoleteProps, boolean hiveEngineEnabled, long maxHiveTablePropertySize, String currentLocation) {
+    Map<String, String> parameters = Optional.ofNullable(tbl.getParameters()).orElseGet(Maps::newHashMap);
```

Comment:
please format as in iceberg
````
Map<String, String> parameters =
    Optional.ofNullable(tbl.getParameters()).orElseGet(Maps::newHashMap);
Map<String, String> summary =
    Optional.ofNullable(metadata.currentSnapshot())
        .map(Snapshot::summary)
        .orElseGet(ImmutableMap::of);
````

Reply:
Formatted as in the snippet above, but can you please explain how it should be formatted going forward?
I thought in Hive we want to have less line breaks as long as it fits line length of 120 characters.